### ISSUE TITLE
[SF][A11y][ui5-textarea]: growingMaxLines does not work when valueState is Error

### ISSUE THREAD

**Issue by i323808** [SAP/ui5-webcomponents#8767]
### Bug Description

growingMaxLines dose not work and only one line text is visible when set ui5-textarea properties as below:
rows = 1
valueState = Error
growing = true
growingMaxLines > 1

### Affected Component

ui5-textarea

### Expected Behaviour

The maximum number of lines should align with the `growingMaxLines` property.

### Steps to Reproduce

1. navigate to https://sap.github.io/ui5-webcomponents/playground/?path=/docs/main-text-area--text-area-overview 
2. set rows = 1, valueState = Error, growing = true, growingMaxLines > 1
3. type text in text area component
4. only one line text is visible when the number of lines is greater than growingMaxLines

### Log Output, Stack Trace or Screenshots
<img width="1163" alt="ui5_textarea_growing" src="https://github.com/SAP/ui5-webcomponents/assets/30744017/32908e11-1b6b-4565-822e-27b7c399e87e">

_No response_

### Priority

None

### UI5 Web Components Version

1.24.0

### Browser

Chrome, Edge, Firefox, Safari

### Operating System

macOS Sonoma 14.3.1

### Declaration

- [X] I’m not disclosing any internal or sensitive information.

**Comment by dimovpetar** [comment]
Hello @ui5-webcomponents-topic-rl,

Seems value state play role here.
I have tested with `growingMaxLines=5`.
![image](https://github.com/SAP/ui5-webcomponents/assets/32839090/3dffe31a-629e-4ba3-b46d-e8bb7f98793b)
Same properties, but with `valueState=Success`:
![image](https://github.com/SAP/ui5-webcomponents/assets/32839090/19288674-4e22-493c-9233-71d7a94fceb3)

Best regards,
Petar
